lib/topology_docker/plugin/__init__.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2015-2016 Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""
+topology module entry point.
+"""
+
+from __future__ import unicode_literals, absolute_import
+from __future__ import print_function, division
+
+__author__ = 'Hewlett Packard Enterprise Development LP'
+__email__ = 'hpe-networking@lists.hp.com'
+__version__ = '1.7.1'
```

Comment:
Why is this?
 - Because the logs are under <node>_topology_uid folder name, which makes it difficult to get the right log when something fails if you are constantly running tests

 What does it does?
 - Based on the test name, it creates a folder with that name and copies the node_topology folders used to execute the test

Why is this required in topology_docker? Why if this is openswitch specific thing why this isn't in topology_docker_openswitch?
 - It may not be required in topology_docker, if everyone agrees I could move it to docker_openswitch